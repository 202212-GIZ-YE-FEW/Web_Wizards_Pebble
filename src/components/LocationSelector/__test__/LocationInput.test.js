import renderer from "react-test-renderer";

import Input from "../LocationInput";

it("renders correctly", () => {
    const tree = renderer.create(<Input />).toJSON();
    expect(tree).toMatchSnapshot();
});
