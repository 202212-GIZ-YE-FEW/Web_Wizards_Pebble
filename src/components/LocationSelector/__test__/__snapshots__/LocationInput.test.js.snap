// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<input
  className="!rounded-md !border-2 !border-black-100 p-2 !bg-white"
  type="text"
/>
`;
